src/joystick/joystick.cc

```diff
@@ -172,4 +174,47 @@ void Joystick::Close() {
   }
 }
 
+size_t Joystick::Remap(size_t num, size_t type){
+  if(mode_ == "Logitech_F710"){
+    // no need to remap
+  }
+  else if (mode_ == "Sony_DualShock_4"){
+    if(type == JS_EVENT_AXIS){
+      switch(num){
+        case 5:
+        case 3:
+          num--;
+          break;
+        case 2:
+        case 4:
```

Comment:
might be nice to add comments labelling what these axes correspond to? I imagine some are arrow keys and some are the left/right joystick?